server/src/main/clients/web-browser/src/libs/projects/ProjectPost/ProjectPost.tsx

```diff
@@ -0,0 +1,21 @@
+import {ReactNode} from 'react';
+
+function ProjectPost(props: {
+  title: string;
+  date: Date;
+  image: string;
+  content: ReactNode;
+}) {
+  return (
+    <article>
```

Comment:
What do you think about using AntD Card? Ex: https://github.com/DaVinciSchools/leo/blob/ceb590689a32d2fd8aec96d2d0c634b4c83b9356/server/src/main/clients/web-browser/src/libs/ProjectCard/ProjectCard.tsx